# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../heimdal/heimdal-1.3.3-linux3-pthread.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 0b3c720c014d6813aa4f14d4800b7e57f34380de Mon Sep 17 00:00:00 2001
From: Jelmer Vernooij <jelmer@samba.org>
Date: Fri, 22 Jul 2011 14:19:34 +0200
Subject: [PATCH] cf: Also enable pthreads on Linux 3.

Signed-off-by: Love Hornquist Astrand <lha@h5l.org>
---
 cf/pthreads.m4 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cf/pthreads.m4 b/cf/pthreads.m4
index 7ce7310..209e4f3 100644
--- a/cf/pthreads.m4
+++ b/cf/pthreads.m4
@@ -41,7 +41,7 @@ case "$host" in
 	;;
 *-*-linux* | *-*-linux-gnu)
 	case `uname -r` in
-	2.*)
+	2.*|3.*)
 		native_pthread_support=yes
 		PTHREAD_CFLAGS=-pthread
 		PTHREAD_LIBADD=-pthread
-- 
1.6.5.GIT

